### PR TITLE
Update SampleDataUtil and One Potential Bug/fix

### PR DESCRIPTION
**Pull request by e0031374**
tl;dr
Bug Replication steps:
1. group find bts
2. contact delete [some bts contactid]

this PR attempts to "fix" it by making contact delete always display all contacts in addresbook after it is done

==== long version ====

Basically, the original AB3 behaviour was that if you did the following
1. contact find something
2. contact delete 3

after these two steps, contact list will still show whatever the list was after step 1 just without the deleted contact instead of the full addressbook list of contacts without the deleted contact

this may cause the unintended problems since groups is now sharing this part
bug replication:
1. group find bts
2. contact delete [some bts contactid]

and you will find that it reverts to contact view but filtered to only show contacts in bts less the deleted contact. (and without the group panel)

so one way around this is to force "contact delete" to always show the full addressbook list after deleting, much like how adding a new contact does right now. this would bring contact delete in line with behaviour with "group delete"
but am i missing something?

### CODE REVIEW

**Comment by tanlk99** [comment]
> tl;dr
> Bug Replication steps:
> 
> 1. group find bts
> 2. contact delete [some bts contactid]
> 
> this PR attempts to "fix" it by making contact delete always display all contacts in addresbook after it is done
> 
> ==== long version ====
> 
> Basically, the original AB3 behaviour was that if you did the following
> 
> 1. contact find something
> 2. contact delete 3
> 
> after these two steps, contact list will still show whatever the list was after step 1 just without the deleted contact instead of the full addressbook list of contacts without the deleted contact
> 
> this may cause the unintended problems since groups is now sharing this part
> bug replication:
> 
> 1. group find bts
> 2. contact delete [some bts contactid]
> 
> and you will find that it reverts to contact view but filtered to only show contacts in bts less the deleted contact. (and without the group panel)
> 
> so one way around this is to force "contact delete" to always show the full addressbook list after deleting, much like how adding a new contact does right now. this would bring contact delete in line with behaviour with "group delete"
> but am i missing something?

Looks correct. Just set "contact delete" to `ViewType.CONTACT`.
